doc/div-alg.xml

```diff
@@ -10,21 +10,13 @@
    <Prop Name="WedderburnDecompositionWithDivAlgParts" 
          Arg="A"/>
    <Returns>
-         A list of lists <C>[r,D]</C>, each representing a ring of 
-         <M> r \times r </M> matrices over a field or division algebra <K>D</K>. 
+         A list of lists <C>[r,D]</C>, each representing a ring of <M> r \times r </M> matrices over a field or division algebra <K>D</K>. 
```

Comment:
Here and below, there seem to be manual sections where the text is reformatted but not changed. This should be reverted. Otherwise, it pollutes the history of changes but mixing together non-related changes in the same commit. If one has to reformat code/text, a separate commit for that is recommended. But here I can not see why the text required reformatting (maybe this is an editor setting, which should rather be switched off?). It is more efficient to split long paragraphs into separate lines, because this allows to see changes easier.